src/index.html

```diff
@@ -1,13 +1,297 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>v2 Nothing</title>
```

Comment:
Consider it will be better
```suggestion
    <title>Nothing</title>
```